src/report.rs

```diff
@@ -44,7 +46,50 @@ impl Report {
       format!("unix:{:?}", platform.os_type).into()
     };
 
-    let backtrace = format!("{:#?}", Backtrace::new());
+    //We skip non-user code frames, including Backtrace::new()
+    const SKIP_FRAMES_NUM: usize = 8;
+    //Code is based on backtrace source
+    const HEX_WIDTH: usize = mem::size_of::<usize>() * 2 + 2;
```

Comment:
I find these two constants confusing. Maybe add a small comment and rationale _why_ they are what they are, instead of just referencing to the authority that produced them